ext/hal/st/lib/sensor/vl53l0x/api/platform/src/vl53l0x_platform.c

```diff
@@ -37,7 +37,15 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "vl53l0x_platform.h"
 #include "vl53l0x_i2c_platform.h"
 #include "vl53l0x_api.h"
-#include <Windows.h>
+#include <sensor.h>

```

Comment:
Is that possible to add a OS abstraction source file for porting this code on Zephyr APIs?